tests/resources/utils-data.ts

```diff
@@ -30,65 +30,270 @@ export const sampleTaxonomy = {
   ],
 } as Taxonomy;
 
-export const sampleFacetTrie = {
-  genres: {
+export const sampleFacetTrie = [
```

Comment:
nitty nit nit: Should we rename this to sampleFacetOption?

Reply:
good catch!! done. 

minor note, i will re add the `Trie` facet and the test. Trie structure and list structure are both stored server side for 1. validation against taxonomy and 2. taxonomy responses, respectively